sregistry/client/pull.py

```diff
@@ -18,32 +18,27 @@ def main(args,parser,subparser):
 
     from sregistry.main import get_client
  
-    images = args.image
-    if not isinstance(images, list):
-        images = [images]
+    image = args.image
+    if not isinstance(image, str):
```

Comment:
I would instead check if it exists, argparse is going to require it to be defined, and a string.

Reply:
Good morning!

You're right! Will fix that!

Reply:
Actually if you do this and don’t use nargs the user will need to specify a flag - I don’t think we want that. It’s better to just require 1 argument and grab it from index 0 in the resulting list.

Reply:
I don't think a flag is necessary: it just expects a single image name. This is what I get now:

```bash
sregistry pull -h
usage: sregistry pull [-h] [--name NAME] [--force] [--no-cache] image

positional arguments:
  image        full uri of image

optional arguments:
  -h, --help   show this help message and exit
  --name NAME  custom name for image
  --force, -f  force overwrite of existing image
  --no-cache   if storage active, don't add the image to it
```

Reply:
Yeah I think you are right, it’s the — that distinguishes!

It’s too early I’m going to go back to sleep for a bit, looking forward to reviewing further when it’s light out! 🌞